### PR TITLE
fix(j-s): Civil claimants national id

### PR DESCRIPTION
**Pull request by oddsson**
# Save civil claimants national id even though its not found in the national registry

[Asana](https://app.asana.com/0/1199153462262248/1208522018913394/f)

## What

We are not saving a civil claimant's national id to database when the national id is not found in the national registry. This PR fixes that. It also refactors the `Toast` component in Island UI. The refactoring includes

- Updating to a newer desing
- Start using the newer `Icon` component instead of the old deprecated one.

## Why

If the national registry is down, we still want to keep the national id set by the prosecutor. 

## Screenshots / Gifs

https://github.com/user-attachments/assets/de32cc4e-eb98-479d-ace0-3ddd74c19867

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
	- Enhanced data handling for civil claimants in the Processing component, improving robustness.

- **Style**
	- Increased width and adjusted padding of toast notifications for better visibility and responsiveness.
	- Repositioned close button for improved user experience in toast notifications.

- **Refactor**
	- Simplified component imports and updated icon management in toast notifications.

### CODE REVIEW

File: libs/island-ui/core/src/lib/Toast/Toast.css.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ globalStyle(`${root} .Toastify__toast-container`, {
   transform: 'translate3d(0, 0, 9999px)',
   position: 'fixed',
   padding: '4px',
-  width: '320px',
+  width: '432px',
```
Comment (Toti91) [inline libs/island-ui/core/src/lib/Toast/Toast.css.ts:17]: Do we need to think about screens smaller than this? F.x. iPhone 12 Pro is 390px

Comment (Toti91) [reply libs/island-ui/core/src/lib/Toast/Toast.css.ts:17]: Ah nvm didn't see the media query below

**Review by Toti91 — APPROVED** [review]
LGTM

**Review by gudjong — APPROVED** [review]

**Comment by thorkellmani** [comment]
Cycled `automerge` since it was blocking the queue 🙇

**Pull request merged**